usr/src/cmd/dd/dd.c

```diff
@@ -1035,6 +1066,12 @@ main(int argc, char **argv)
 
 			ibc = read(ibf, (char *)ibuf, ibs);
 
+			if (istriden > 0 && lseek(ibf, istriden * ((off_t)ibs),
```

Comment:
Mostly I just look for potential overflow issues in new code as we're writing it as opposed to trying to retrofit everything. In this case, if we were going to overflow, I'd probably error out rather than trying to lseek to garbage. I'm not sure what makes the most sense here in this case, especially as their both signed values so overflow is undefined. I don't want to hold things back based on this, but hopefully for other stuff going forward where we're introducing the cases we can make sure to handle it.